core/shared/src/main/scala/org/http4s/Message.scala

```diff
@@ -764,7 +763,7 @@ object Response extends KleisliSyntax {
   val notFound: Response[Pure] =
     Response(
       Status.NotFound,
-      entity = Entity(Stream("Not found").through(utf8.encode)),
+      entity = Entity.Strict(Chunk.array("Not found".getBytes(StandardCharsets.UTF_8))),
```

Comment:
A strict UTF8 string seems a common pattern, I wonder if it deserves a special helper constructor.

Reply:
Adding in another PR.